UPGRADE.md

```diff
@@ -51,6 +51,12 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 
 ## [From 7.0.0-beta2 to Unreleased]
 ### [shopsys/framework]
+- [#627 model service layer removal](https://github.com/shopsys/shopsys/pull/627)
+    - we moved business logic into entities, following instructions tells you what method was moved where, so you can upgrade your code and also tests.
+      If you use these methods change their calling appropriately:
+        - `CustomerService::edit(User $user, UserData $userData)`  
+          ⮕ `User::edit(UserData $userData, CustomerPasswordService $customerPasswordService)`
```

Comment:
In your document, I saw that you are planning to remove `CustomerPasswordService`...It might be confusing later if you do not modify the upgrade notes :slightly_smiling_face:

Reply:
I'm planning to update them (this situation occured because we wanted to try difficult service, and unfortunatelly it depended on another service), but thanks for noticing